src/main/java/duke/task/ToDo.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+public class ToDo extends Task {
+    /**
+     * Constructor Method for ToDo.
+     *
+     * @param title title of the ToDo.
+     */
+    public ToDo(String title) {
+        super(title);
+        this.type = "[T]";
+    }
+
+    /**
+     * Overloaded Constructor for when loading from hard drive.
+     *
+     * @param title title of the ToDo.
+     * @param done boolean to represent if task is done or not;
+     */
+    public ToDo(String title, boolean done) {
+        super(title);
+        this.type = "[T]";
+        this.done = done;
+    }
```

Comment:
I'm not sure what this does, may I know how it works?

Reply:
Yep sure! This is an overloaded constructor for when I read the data from the stored data. I need it for setting the status of the task on initialization since my original constructor would default to not done.